app/Http/Middleware/CamelizeJsonResponse.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright (c) 2018 Adshares sp. z o.o.
+ *
+ * This file is part of AdServer
+ *
+ * AdServer is free software: you can redistribute and/or modify it
+ * under the terms of the GNU General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * AdServer is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with AdServer. If not, see <https://www.gnu.org/licenses/>
+ */
+
+namespace Adshares\Adserver\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\JsonResponse;
+
+class CamelizeJsonResponse
+{
+    public function handle($request, Closure $next)
+    {
+        $response = $next($request);
+        
+        if ($response instanceof JsonResponse) {
+            return $this->camelizeJsonResponse($response);
+        }
+
+        return $response;
+    }
+
+    private function camelizeJsonResponse(JsonResponse $response)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this function name "camelizeJsonResponse" with "__construct". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1603)